projects/plugins/crm/src/js/components/automations-admin/components/edit-modal/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { Button } from '@automattic/jetpack-components';
+import { Modal } from '@wordpress/components';
+import { sprintf, __ } from '@wordpress/i18n';
+import { useCallback } from 'react';
+import { StepConfig } from '../step-config';
+import styles from './styles.module.scss';
+import type { Step, Workflow } from 'crm/state/automations-admin/types';
+
+type EditModalProps = {
+	workflow: Workflow;
+	isOpen: boolean;
+	onClose: () => void;
+};
+
+export const EditModal: React.FC< EditModalProps > = ( { isOpen, onClose, workflow } ) => {
+	const steps: Step[] = [];
+
+	let stepId: string | undefined = workflow.initial_step;
+	while ( stepId ) {
+		const step: Step = workflow.steps?.[ stepId ];
+		steps.push( step );
+		stepId = step.next_step;
+	}
+
+	const onSave = useCallback( () => {
+		// TODO: save workflow here
+		onClose();
+	}, [ onClose ] );
+
+	const onCancel = useCallback( () => {
+		// TODO: reload workflow here
+		onClose();
+	}, [ onClose ] );
+
+	return (
+		isOpen && (
+			<Modal onRequestClose={ () => onClose() }>
+				<div className={ styles.container }>
+					<h1>
+						{ sprintf(
+							/* translators: placeholder is a string which is used to identify the workflow */
+							__( 'Edit %1$s', 'zero-bs-crm' ),
+							workflow.name
+						) }
+					</h1>
```

Comment:
From what I can see in the Figma file, then the title is supposed to be `20px`, but it seems to be `36px` fron the default styling of `<h1>`